base58/base58.go

```diff
@@ -90,9 +89,11 @@ func Encode(b []byte) string {
 	x := new(big.Int)
 	x.SetBytes(b)
 
-	answer := make([]byte, 0, len(b)*136/100)
+	// maximum length of output is log58(2^(8*len(b))) == len(b) * 8 / log(58)
+	maxlen := int(float64(len(b))*1.365658237309761) + 1
```

Comment:
You could probably get away with a `float32` here, which is still significantly bigger than what `len(b)` could ever be. Thoughts?

Reply:
My 2c: Doesn't really matter. There'll never be a detectable performance impact, it's equally readable either way. Not worth changing.